### PR TITLE
Build Docker Image

### PR DESCRIPTION
**Pull request by liuhongchao**
The images is currently pushed at liuhongchao/gpu-miner:v0.6

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM nvidia/cuda:11.0-devel-ubuntu20.04 AS builder
+
+WORKDIR /src
+
+RUN apt update
+RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
+RUN apt install -y libuv1-dev
+
+COPY . /src
+RUN make gpu
```
Comment (polarker) [inline Dockerfile:10]: Just an idea for the future optimization:

Is it possible to add only the source code in the docker image without building it ? As the performance might be driver-dependent, building the miner on demand should be more performant

Comment (liuhongchao) [reply Dockerfile:10]: Do you mean the user of this image will then build it from the source code from within the docker image somehow?

**Review by killerwhile — APPROVED** [review]
LGTM. Nicely done!

File: Dockerfile (hunk 1 above)
Comment (polarker) [reply Dockerfile:10]: yes. The docker image contains the source code as a zip file. Whenever user start the docker container, docker will: 1. unzip the source code; 2. make gpu; 3 run gpu-miner

Comment (polarker) [reply Dockerfile:10]: Let's forgot about this due to the following 2 reasons:
1. the performance looks great with pre-built binary
2. if we want to build the binary inside the container, we will need to install cuda toolkit which will take time and space

How do you think ?

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**